client/task-list/tasks/payments/PaymentGatewaySuggestions/components/List/Item.js

```diff
@@ -18,26 +18,27 @@ import './List.scss';
 export const Item = ( {
 	isRecommended,
 	markConfigured,
-	paymentGateways,
-	suggestion,
-	suggestionKeys,
+	paymentGateway,
+	gatewayIds,
 } ) => {
-	const { image, content, id, plugins = [], title, loading } = suggestion;
+	const {
+		image,
+		content,
+		id,
+		plugins = [],
+		title,
+		loading,
+		enabled: isEnabled = false,
+		needsSetup = false,
+		requiredSettingsKeys = [],
```

Comment:
Should this be updated to `requiredSettings`?

Reply:
Yup! Good catch.